src/unitxt/operators.py

```diff
@@ -2104,3 +2225,13 @@ def verify(self):
                 f"If given, duplication_index_field must be a string. "
                 f"Got: {self.duplication_index_field}"
             )
+
+
+class DeleteSplits(MultiStreamOperator):
```

Comment:
Thank for adding these operators. They will be useful in many case.
I think we need to add tests for the new operators.